src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -440,8 +547,24 @@ function ReportActionItemMessageEdit(
                             selection={selection}
                             onSelectionChange={(e) => setSelection(e.nativeEvent.selection)}
                             isGroupPolicyReport={isGroupPolicyReport}
+                            shouldCalculateCaretPosition
+                            onScroll={onSaveScrollAndHideSuggestionMenu}
                         />
                     </View>
+
+                    <Suggestions
+                        ref={suggestionsRef}
+                        isComposerFocused={textInputRef.current?.isFocused()}
+                        updateComment={updateDraft}
+                        measureParentContainerAndReportCursor={measureParentContainerAndReportCursor}
+                        isGroupPolicyReport={false}
+                        // Input
```

Comment:
What's the purpose of this comment?

Reply:
It appears after some refactoring in the suggestion component for the main composer, so I did the same for the edit input composer.
However, since we don't have guidelines for that, I'll remove it